core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -80,10 +80,10 @@ private static class PrefixProps {
    * @return property value
    */
   public String get(String property) {
-    try {
-      return get(Property.valueOf(property));
-    } catch (IllegalArgumentException e) {
-      // Could be a client or custom property, fall back to filtering
+    Property p = Property.getPropertyByKey(property);
+    if (p != null) {
+      return get(p);
+    } else {
```

Comment:
Style-wise, you could probably just remove the "else" line and corresponding curlies, and move the contents of that block up one level of indentation.

Reply:
Yeah, but I think at first glance it's easier to read this way.